### DISCUSSION TITLE
alpine expression error when using wire:scroll and h-full tailwind class

### DISCUSSION THREAD

**Discussion by alxndrmlr** [livewire/livewire#8217] [7. Bugs (archived)] [answered]
### Livewire version

v3.4.9

### Laravel version

v10.47.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Look at the simple code for this wirebox link https://wirebox.app/b/47mme and visit the hosted version in Chrome with the dev tools open at https://47mme.wirebox.app:8443
2. Scroll the page, and see the console error `livewire.js?id=5d8beb2e:1 Alpine Expression Error: Unexpected token '}' Expression: "$wire."`
3. Fork the wirebox and remove the `h-full` class (or load the page and remove the `h-full` class before you scroll page). If you do this no error will occur

### Repository or Wirebox URL

https://wirebox.app/b/47mme

### Additional information

I ran into this will building an infinite scroll experience for my app, and found I could reproduce when this simple example. It seems odd to me that the `h-full` class being added trips Livewire (or rather the alpine code underneath).

Let me know if you need more detail on this, or anything else I can add to this to help trace it down. I've been able to source dive into Livewire code to try to find issues but I'm out of my depth in this Javascript (hence why I love to use alpine/livewire).

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by rafwoski** [comment]
Hi,
i ran into the same problem. Everything seems to be working fine but console shows exactly the same error.  Did you manage to fix it?

**Comment by hoanghjx** [comment]
+1

**Comment by faisuc** [comment]
@alxndrmlr @rafwoski  I've created a PR #9586 to fix this issue. It's currently in draft. Could you check if it works on your end?

**Comment by calebporzio** [comment, marked as answer]
Just PR'd this to main branch: https://github.com/livewire/livewire/pull/9605

Empty wire:* expressions will now not throw an error 👌

Also I changed `wire:scroll` to `wire:navigate:scroll` to not cause confusion
